src/components/Table/Table.scss

```diff
@@ -33,6 +33,7 @@ $border-color: color(sky, base);
     .subText {
       margin-top: 2px !important;
       margin-bottom: 0px !important;
+      word-break: break-all;
```

Comment:
@Farhan-emgage I don't think this will work, as it will even break small words instead of wrapping them. Try overflow-wrap: break-word;